### ISSUE TITLE
Why does the learning rate remain the same

### ISSUE THREAD

**Issue by yzh1527** [AlexeyAB/darknet#5840]
train commend:
./darknet detector train BDD100k/bdd100k.data cfg/yolov3_BDD.cfg darknet53.conv.74 -gpus 0,1,2,3  > log/BDD61.log

yolov3_BDD.cfg:
learning_rate=0.00025
burn_in=1000   
max_batches = 100000
policy=steps   
steps=20000,50000,70000
scales=.1,.1,.1
log:
25092: 6.897747, 7.356840 avg loss, 0.001000 rate, 2.199898 seconds, 2007360 images, 33.032042 hours left
Loaded: 0.000086 seconds
Syncing... Done!

 25096: 5.999538, 7.221110 avg loss, 0.001000 rate, 2.134999 seconds, 2007680 images, 33.159485 hours left
Loaded: 0.000079 seconds
Syncing... Done!

 25100: 6.373835, 7.136382 avg loss, 0.001000 rate, 2.169570 seconds, 2008000 images, 33.272133 hours left
Loaded: 0.000114 seconds
Syncing... Done!

 25104: 6.347554, 7.057499 avg loss, 0.001000 rate, 2.175121 seconds, 2008320 images, 33.390823 hours left
Loaded: 0.000063 seconds

Why does the learning rate remain the same   0.001????

**Comment by AlexeyAB** [comment]
https://github.com/pjreddie/darknet/blame/master/examples/detector.c#L27

This is the legacy of the original repository. I will have to think about it.

**Comment by yzh1527** [comment]
yes ,I know learning_rate = 0.00025 * 4=0.001
but 
max_batches = 100000
policy=steps
steps=20000,50000,70000
scales=.1,.1,.1

when the step is 20000,the learning_rate should scales .1,why remain 0.001,isnot 0.0001???

log:
25092: 6.897747, 7.356840 avg loss, 0.001000 rate, 2.199898 seconds, 2007360 images, 33.032042 hours left
Loaded: 0.000086 seconds
Syncing... Done!
25096: 5.999538, 7.221110 avg loss, 0.001000 rate, 2.134999 seconds, 2007680 images, 33.159485 hours left
Loaded: 0.000079 seconds
Syncing... Done!
25100: 6.373835, 7.136382 avg loss, 0.001000 rate, 2.169570 seconds, 2008000 images, 33.272133 hours left
Loaded: 0.000114 seconds
Syncing... Done!
25104: 6.347554, 7.057499 avg loss, 0.001000 rate, 2.175121 seconds, 2008320 images, 33.390823 hours left
Loaded: 0.000063 seconds

**Comment by AlexeyAB** [comment]
Attach your cfg-file.

**Comment by yzh1527** [comment]
this is my cfg-file.
thanks
[yolov3_BDD.cfg.txt](https://github.com/AlexeyAB/darknet/files/4730127/yolov3_BDD.cfg.txt)

**Comment by KelvinHuang666** [comment]
@AlexeyAB I have the same question that learning rate is not on the decrease with thaining.

**Comment by KelvinHuang666** [comment]
@AlexeyAB I have the same question that learning rate is not on the decrease with training.
